include/powsybl/iidm/NetworkVariant.hpp

```diff
@@ -40,6 +44,10 @@ class VariantImpl : public Variant<Network, VariantImpl> {
 
     VariantImpl& operator=(VariantImpl&&) noexcept = delete;
 
+    const BusCache& getBusBreakerViewCache() const;
```

Comment:
To be discussed: this is necessary because cache must be available from BusBreaker*Impl (so that getBus() is called on. IMO that would be ugly to add friendship for those impl classes...